Lab2.ipynb

```diff
@@ -0,0 +1,879 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "pois_tf = poisson(0.5)\n",
+    "for i in range(len(pixels_seq)):\n",
+    "    dets[i] = pois_tf.ppf(1-sigma5/pixels_seq[i])\n",
+    "fig, ax = plt.subplots(1, 1)\n",
+    "ax.plot(pixels_seq, dets, 'bo', lw=2, alpha=0.6, label='norm 10000 pdf')\n",
+    "plt.show()"
```

Comment:
I'm wondering why there is a flat stage in the middle of the graph? I got the same pattern as you, so this question is just for more thoughts to investigate later

Reply:
I think the flat stage is because poisson distribution is a discrete distribution. The flat stage means at those x values, the detections are not changing.